app/helpers.php

```diff
@@ -137,3 +137,10 @@ function has_include($request, $include)
 
     return false;
 }
+
+function convert_to_date($value, $timezone = 'UTC')
```

Comment:
Oh sorry, one last thing –– let's add a docblock to this function!

Reply:
Would it make sense to default to 'America/New_York' instead of `UTC`?

Reply:
I think `UTC` makes the most sense as a default, since that's the timezone our dates are stored in. Converting this date to be the beginning of the day in a different timezone is a specific case.